app/views/events/_form.html.erb

```diff
@@ -17,17 +17,21 @@
       <%= form.label :title %><br>
       <%= form.text_field :title, class: "form-control" %>
     </div>
-    <div class="custom-file">
-      <%= form.label "Upload a logo", class: "custom-file-label" %>
-      <%= form.file_field :logo, class: "custom-file-input", accept: 'image/jpeg, image/png'  %>
+
+    <div data-controller="preview">
+      <div class="custom-file" data-controller="preview">
+        <%= form.label "Upload a logo", class: "custom-file-label", id: "logo-label"%>
+        <%= form.file_field :logo, class: "custom-file-input", accept: 'image/jpeg, image/png', id: "image-path", "data-action" => "change->preview#show" %>
```

Comment:
use latest syntax `"data-action": "change->preview#show"`
using `=>` is old syntax.